modules/data-model/forms/impl/src/main/java/io/uhndata/cards/forms/internal/serialize/FormAnswerFilterProcessor.java

```diff
@@ -73,6 +74,26 @@ public String getName()
         return "answerFilter";
     }
 
+    @Override
+    public String getDescription()
+    {
+        return "Only runs on forms.\n"
+            + "Include or exclude answers or answer sections based on the question or section they refer to.\n"
+            + "If only include options are provided, only those nodes and any descendants will be included\n"
+            + "If only exclude options are provided, all other nodes will be included.\n"
+            + "If both include and exclude options are provided, then for a node to be included it must be a "
+            + "specified include node or descendent thereof and also not be a member or descendant of an exclude node."
+            + "\nIt is not possible to include a descendant of an excluded node";
+    }
+
+    @Override
+    public SelectorDetails getDetails()
+    {
+        return new SelectorDetails(getName(), getDescription(), isEnabledByDefault(null),
+            "include", "A path to an included node. `answerFilter:include=/Questionnaires/Path/To/Question`",
+            "exclude", "A path to an excluded node. answerFilter:exclude=/Questionnaires/Path/To/Question");
```

Comment:
To check of the lines aren't too long for the checkstyle plugin:
```suggestion
            "include", "A path to an included question or section. `answerFilter:include=/Questionnaires/Path/To/Question`",
            "exclude", "A path to an excluded question or section. answerFilter:exclude=/Questionnaires/Path/To/Question");
```